zipkin-server/src/main/java/zipkin/server/ZipkinQueryApiV1.java

```diff
@@ -43,6 +44,7 @@
  */
 @RestController
 @RequestMapping("/api/v1")
+@CrossOrigin("${zipkin.query.allowed-origins:'*'}")
```

Comment:
yeah src/it would be ideal!

On Wed, Aug 17, 2016 at 9:03 AM, Ho Yan Leung <notifications@github.com>
wrote:

> In zipkin-server/src/main/java/zipkin/server/ZipkinQueryApiV1.java
> <https://github.com/openzipkin/zipkin/pull/1234#discussion_r75048464>:
>
> > @@ -43,6 +44,7 @@
> >   */
> >  @RestController
> >  @RequestMapping("/api/v1")
> > +@CrossOrigin("${zipkin.query.allowed-origins:'*'}")
>
> Yeah the single quotes aren't needed - removed them.
>
> Happy to try and get some ITs in there - are the ones in
> zipkin-server/src/it a good example to start with?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openzipkin/zipkin/pull/1234/files/87ecb3a27cd70ad4ec42cc432532cee40f561a82#r75048464>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAD619LSQdcAAAtv2WfFKbQccJGabT2xks5qgl37gaJpZM4JhLqg>
> .
>